### PR TITLE
For deletion

### PR DESCRIPTION
**Pull request by Y2nn1c**
Trash

### CODE REVIEW

**Comment by cl8n** [comment]
the "SKIP_OUTDATED_CHECK" thing is for when you have a specific reason to bypass it (if you're not yet familiar with the details of this system, basically just don't write this)

**Comment by Y2nn1c** [comment]
> the "SKIP_OUTDATED_CHECK" thing is for when you have a specific reason to bypass it (if you're not yet familiar with the details of this system, basically just don't write this)

oh ok, sry I thought it was meant for translations only so I used since I didn't change any words themselves.
So I just add the thing at the top of the file on all the other languages and thats it?

File: wiki/Main_page/en.md
Comment (cl8n) [inline wiki/Main_page/en.md:108]: same as above

**Review by cl8n — CHANGES_REQUESTED** [review]

File: wiki/Main_page/en.md
Comment (Y2nn1c) [reply wiki/Main_page/en.md:20]: The sitemap was also under "Wiki and meta pages" and I thought I would leave it there and take it out of "Getting Started" since this huge list doesn't seem to serve the purpose to help out new players.
I moved Bancho Bot to the "Game Client" section since there should still be a way to get to the Bancho Bot page.

File: wiki/Main_page/en.md
Comment (Y2nn1c) [reply wiki/Main_page/en.md:108]: I did both splits since those • were at the end of the lines. So now it looks the same but the dot isn't at the end anymore.
I thought this dot at the end of a line looks bad but I can put it back there if you wish so

File: wiki/Main_page/en.md
Comment (Y2nn1c) [reply wiki/Main_page/en.md:27]: Applied it, wonderful job as always sir o7

File: wiki/Main_page/en.md
Comment (cl8n) [reply wiki/Main_page/en.md:110]: making this change won't fix, or may make even worse, the views on mobile, lazer, etc., there is not one fixed display of text that we can rely on to make purely visual changes like this unfortunately. so for now just revert

Comment (Y2nn1c) [reply wiki/Main_page/en.md:110]: okay

**Comment by Y2nn1c** [comment]
@cl8n I'm done so far, if you are fine with the english version, do I only put the tag for outdated translation to the other languages if required and am done with it?

**Comment by The-Last-Cookie** [comment]
Regarding the whole article, we should discuss if we want to keep the old embellishments addressing the user directly in a very sprawling way or if we use a more plain succinct general description thing.

Personally, I liked the phrase `and turn your ideas into reality` what you just had in one of your commits. Also, some German translations fit really well on the main page but that's just me.

**Comment by Y2nn1c** [comment]
> Regarding the whole article, we should discuss if we want to keep the old embellishments addressing the user directly in a very sprawling way or if we use a more plain succinct general description thing (or some sort of in-between).
> 
> Personally, I liked the phrase `and turn your ideas into reality` what you just had in one of your commits. Also, some German translations fit really well on the main page but that's just me.

Tbh I thought it sounded nice but at the other hand that's not something to put into a proper main page. We already have an abnormal amount of text in it, since it should normally just be a few short sentences so I think it's right to make it as short and accurate as I can. Also what translations do you mean?

**Comment by The-Last-Cookie** [comment]
Both implementations (embellishments on the one, and succinct texts on the other) have their up and down sides.

I don't know how to put it really. I guess that it's just what I would like to see in the main page personally.

Finding a balance between including every important information in the text and still address the reader directly in some way or another to make the article less formal **is not easy, but it is the goal** in my opinion.

I will leave the final decision to the maintainers and get an impression of the article afterwards.

**Comment by cl8n** [comment]
> @cl8n I'm done so far, if you are fine with the english version, do I only put the tag for outdated translation to the other languages if required and am done with it?

unless you're doing something weird regarding translations (which you aren't, in this PR), you can copy and paste the thing CI tells you into each of the article's translations. or you can let a maintainer do that part for you when they're ready to merge, doesn't rly matter

i thought the changes to the links were good but I haven't reviewed the further rewording yet since u got ur account back

**Comment by Y2nn1c** [comment]
> > @cl8n I'm done so far, if you are fine with the english version, do I only put the tag for outdated translation to the other languages if required and am done with it?
> 
> unless you're doing something weird regarding translations (which you aren't, in this PR), you can copy and paste the thing CI tells you into each of the article's translations. or you can let a maintainer do that part for you when they're ready to merge, doesn't rly matter
> 
> i thought the changes to the links were good but I haven't reviewed the further rewording yet since u got ur account back

Not planning to change anything, so no translated content here. This is the final version if there are no issues that others bring up.

**Pull request closed without merging**
